### PR TITLE
chore: Convert spa/fetch-body-propagation.browser.js

### PR DESCRIPTION
**Pull request by cwli24**
Please add a one-paragraph summary here, suitable for a release notes description. This will help with documentation.
---

### Overview

https://new-relic.atlassian.net/browse/NR-138446

### Testing

All browser wdio: https://github.com/newrelic/newrelic-browser-agent/actions/runs/9038310595 ✅

### CODE REVIEW

**Comment by cwli24** [comment]
All browser WDIO: https://github.com/newrelic/newrelic-browser-agent/actions/runs/9038310595

File: tests/specs/spa/interaction-tracking.e2e.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { notIE } from '../../../tools/browser-matcher/common-matchers.mjs'
 
-describe('interaction tracking works inside', () => {
-  it.withBrowsersMatching(notIE)('single fetch with formData', async () => {
-    const url = await browser.testHandle.assetURL('spa/fetch-formdata-onclick.html')
+describe('interaction tracking', () => {
+  it.withBrowsersMatching(notIE)('works with Request formData', async () => {
```
Comment (metal-messiah) [inline tests/specs/spa/interaction-tracking.e2e.js:4]: can we keep a note somewhere as to why IE is excluded? I am assuming its due to fetch being invoked by formData() after looking through it and by what the test was named originally

File: tests/specs/spa/interaction-tracking.e2e.js
Comment (metal-messiah) [inline tests/specs/spa/interaction-tracking.e2e.js:1]: I also noticed this file does not have an equivalent to the `exceeding max SPA nodes` test from the browser test.  Is that covered somewhere else?

**Comment by patrickhousley** [comment]
I was already working on this test file

File: tests/specs/spa/interaction-tracking.e2e.js (hunk 1 above)
Comment (cwli24) [reply tests/specs/spa/interaction-tracking.e2e.js:4]: IE doesn't support fetch

File: tests/specs/spa/interaction-tracking.e2e.js
Comment (cwli24) [reply tests/specs/spa/interaction-tracking.e2e.js:1]: There's a "nodes in SPA interaction does not exceed set limit" test in functional/spa/basic.test.js doing the exact same thing

**Comment by patrickhousley** [comment]
Included in #1034

**Pull request closed without merging**